### ISSUE TITLE
Beta page updates

### ISSUE THREAD

**Issue by dlevineBC** [bcgov/design-system#123]
- Remove static image example of header
- Accessibility content still references checkboxes "This **checkbox** has been built according to WCAG 2.0 AA standards and all government services should strive to meet this level."
- CSS code is missing the Beta-PhaseBanner class 
- Include "Last Updated" date below the status badge at top
